app/game-session.coffee

```diff
@@ -1,6 +1,6 @@
 {nx} = require 'nexus-node'
 
-RoundPhase    = require 'cssqd-shared/models/round-phase'
+RoundPhase    = require '../shared/models/round-phase'
```

Comment:
seems like it [should work](https://docs.npmjs.com/files/package.json#local-paths) and it works fine for me